components/suggestion/command_provider/app_command_parser/app_command_parser.ts

```diff
@@ -1599,7 +1609,7 @@ export class AppCommandParser {
 
     // getFlagNameSuggestions returns suggestions for flag names
     private getFlagNameSuggestions = (parsed: ParsedCommand): AutocompleteSuggestion[] => {
-        if (!parsed.form || !parsed.form.fields || !parsed.form.fields.length) {
+        if (!parsed.resolvedForm || !parsed.resolvedForm.fields || !parsed.resolvedForm.fields.length) {
```

Comment:
Would this also work?

```suggestion
        if (!parsed?.resolvedForm?.fields?.length) {
```